packages/mirror/src/Mirror.ts

```diff
@@ -131,6 +131,46 @@ export const getProjectFees = async (
   return platformFee + nftPrice
 }
 
+export const getFees = async (
```

Comment:
No issues, this looks good to me. 

Just want to point out that you can combine the multiple reads using viems multicall.

https://viem.sh/docs/contract/multicall#multicall